Source/dvlnet/tcp_client.cpp

```diff
@@ -28,10 +29,40 @@ int tcp_client::join(std::string addrstr)
 	constexpr int MsSleep = 10;
 	constexpr int NoSleep = 250;
 
-	std::string port = StrCat(*sgOptions.Network.port);
+	const char *defaultPort = "6112";

```

Comment:
Should the argument be an `std::string_view` rather than a `std::string` perhaps?
If not, might make sense to assign it to a string view to simplify some slicing below

Reply:
It appears to me that `std::string_view` should work fine. Though I'll have to tweak a bunch of files to do it.